.github/workflows/piper.yml

```diff
@@ -0,0 +1,64 @@
+name: Piper Example
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    paths:
+      - ".github/workflows/piper.yml"
+      - "wasmedge-piper/**"
+  pull_request:
+    paths:
+      - ".github/workflows/piper.yml"
+      - "wasmedge-piper/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install Dependencies for building WasmEdge
+        run: |
+          sudo apt-get update
+          sudo apt-get install liblld-14-dev ninja-build
+
+      - name: Checkout WasmEdge
+        uses: actions/checkout@v4
+        with:
+          repository: WasmEdge/WasmEdge
+          path: WasmEdge
+
+      - name: Build WasmEdge with WASI-NN Piper plugin
+        run: |
+          cmake -GNinja -Bbuild -DCMAKE_BUILD_TYPE=Release -DWASMEDGE_PLUGIN_WASI_NN_BACKEND=Piper
```

Comment:
We don't need to install `liblld-14-dev` here by setting the CMake option `-DWASMEDGE_USE_LLVM=OFF` to disable all AOT-related components.